Documentation/templates/fonts/swift4.md

```diff
@@ -17,9 +17,10 @@ You can customize some elements of this template by overriding the following par
 
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
-| `bundle` | `BundleToken.bundle` | Allows you to set from which bundle font files are loaded from. By default, it'll point to the same bundle as where the generated code is. |
+| `bundle` | `BundleToken.bundle` | Allows you to set from which bundle font files are loaded from. By default, it'll point to the same bundle as where the generated code is. Note: ignored if `lookupFunction` parameter is set. |
 | `enumName` | `FontFamily` | Allows you to change the name of the generated `enum` containing all font families. |
 | `fontTypeName` | `FontConvertible` | Allows you to change the name of the struct type representing a font. |
+| `lookupFunction` | N/A | Allows you to set your own custom lookup function. The function needs to have as signature: `(name: String, family: String, path: String) -> URL?`. The parameters of your function can have any name (or even no external name), but if it has named parameters, you must provide the complete function signature, including those named parameters – e.g. `myFontFinder(name:family:path:)`. Note: if you define this parameter, the `bundle` parameter will be ignored. |
```

Comment:
Do we want to explicitly indicate what the default value for `lookupFunction` is? I mean, technically it is indeed N/A since that's what we test in the templates (if it's not defined, then use X, other use the defined value) but it could help people understand what is expected here if we provide the function that our templates use if the user doesn't provide a `lookupFunction` there?

If it takes too much space in the "Default Value" column and you fear it might be confusing to provide this info in the "Default Value" column anyway (because we also mention "if you define this parameter" in the explanation but saying it has a default value might make people think it always has a defined value…) it could just be a footnote.

For example we could write it that way in the table:
```
| `lookupFunction` | N/A¹ | Allows you to …
```
then below the table put the footnote like:
```md
_(1) if you don't provide a `lookupFunction`, we will call `url(forResource: path, withExtension:)` on the `bundle` parameter instead_
```

By this note I'm hoping that people will understand better what thing their own `lookupFunction` is supposed to replace.

Reply:
Too bad GitHub doesn't support the (unofficial) footnote syntax. I've tried copying the wikipedia look&feel a bit (with a numbered list)